status-im/status-react
# Links preview added to desktop app

fixes #7176

### Summary:

Links preview added to desktop.

![screen shot 2018-12-24 at 7 18 59 pm](https://user-images.githubusercontent.com/5786310/50404573-2f42f080-07b2-11e9-9bf4-2682a09ddec1.png)


### Review notes (optional):
This branch depends on fixes in `react-native-desktop` that aren't yet in master. So before this PR, the [one from react-native-desktop](https://github.com/status-im/react-native-desktop/pull/438) should be merged.

### Testing notes (optional):

#### Platforms (optional)
- macOS
- Linux
- Windows


#### Areas that maybe impacted (optional)
**Functional**
- 1-1 chats
- public chats
- group chats


### Steps to test:
- Open Status
- send link to any chat
- make sure preview appears in a chat

status: ready
